packages/services/package.json

```diff
@@ -1,16 +1,27 @@
 {
   "name": "@penumbra-zone/services",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
These packages might have strange version numbers, but that's just because I've been publishing to test it out. Don't think we need to start on a clean 1.0.0 anyway.